source/override-properties.d.ts

```diff
@@ -0,0 +1,24 @@
+import type {Merge} from './merge';
+
+/**
+Override existing property types in TOriginal with the types in TOverride.
+
+This is useful when you want to override existing properties with a different type and make sure that these properties really exist in TOriginal.
+
+@example
+```
+type Foo = {
+	a: string
+	b: string
+}
+type Bar = OverrideProperties<Foo, {b: number}>
+//=> {a: string, b: number}
+
+type Baz = OverrideProperties<Foo, {c: number}>
+// error TS2559: Type '{c: number}' has no properties in common with type 'Partial{a: unknown; b: unknown}>'.
+```
+*/
```

Comment:
You need to add categories. See other types for example.

Reply:
I didn't see that, thanks for pointing out